examples/Training/java/src/training/RawDataAccess.java

```diff
@@ -141,7 +141,9 @@ private void retrieveTile()
             }
         } catch (Exception e) {
             throw new Exception("Cannot read the tiles", e);
-        } 
+        } finally {
+            rdf.close();
```

Comment:
Relevant to https://trello.com/c/K4d0rFQc/712-try-with-resources-for-blitz-services perhaps.